intl/en/messages.json

```diff
@@ -0,0 +1,64 @@
+{
+
+    "Invalid usage (near option `{option}`), try {{`{program} --help`}}.":
+    "Invalid usage (near option `{option}`), try {{`{program} --help`}}.",
+    "Invalid usage, try {{`{program} --help`}}.":
+    "Invalid usage, try {{`{program} --help`}}.",
+    "Unable to detect service name, {{package.json}} has no 'name' property.\nPlease update your package.json or specify a service name.\nSee {{`{program} --help`}} for more details.":
+    "Unable to detect service name, {{package.json}} has no 'name' property.\nPlease update your package.json or specify a service name.\nSee {{`{program} --help`}} for more details.",
+    "Error setting up tunnel:":
+    "Error setting up tunnel:",
+    "Failed to deploy `{branchOrPack}` as `{svc}` to `{baseUrl}` via `{exitUrl}`":
+    "Failed to deploy `{branchOrPack}` as `{svc}` to `{baseUrl}` via `{exitUrl}`",
+    "Deployed `{branchOrPack}` as `{svc}` to `{baseUrl}`":
+    "Deployed `{branchOrPack}` as `{svc}` to `{baseUrl}`",
+
+    "Failed to connect to {url}: {err}":
+    "Failed to connect to {url}: {err}",
+    "Cannot access remote.  If authentication is required, credentials should be given in the URL.":
+    "Cannot access remote.  If authentication is required, credentials should be given in the URL.",
+    "Error connecting to server: {err}":
+    "Error connecting to server: {err}",
+
+    "Directory {workingDir} does not exist":
+    "Directory {workingDir} does not exist",
+    "This directory does not contain a valid {{git}} repository":
+    "This directory does not contain a valid {{git}} repository",
+    "Branch `{branchName}` is not available in this repository":
+    "Branch `{branchName}` is not available in this repository",
+    "invalid branch":
+    "invalid branch",
+    "URL `{url}` is not valid":
+    "URL `{url}` is not valid",
+    "invalid url":
+    "invalid url",
+    "{{git push}} failed":
+    "{{git push}} failed",
+
+    "Invalid URL `{baseUrl}`. Only http:// URLs are supported.":
+    "Invalid URL `{baseUrl}`. Only http:// URLs are supported.",
+    "Invalid URL `{baseUrl}`. Only {{localhost}} URLs are supported.":
+    "Invalid URL `{baseUrl}`. Only {{localhost}} URLs are supported.",
+    "Cannot deploy `{what}`: not an {{npm}} package directory":
+    "Cannot deploy `{what}`: not an {{npm}} package directory",
+    "Invalid path, not a directory":
+    "Invalid path, not a directory",
+    "{{HTTP}} error {statusCode}":
+    "{{HTTP}} error {statusCode}",
+    "Deploy `{what}` to `{baseUrl}` failed: {errMsg}":
+    "Deploy `{what}` to `{baseUrl}` failed: {errMsg}",
+
+    "Error while reading file: {err}":
+    "Error while reading file: {err}",
+    "Invalid URL `{baseUrl}`. Only http:// URLs are supported.":
```

Comment:
it is.  slt-globalize lint detects duplicate keys across multiple JSON files, but not within a JSON file.  JSON.parse overwrites and keeps the last one if the values are different.